functions/src/process-email-queue.ts

```diff
@@ -0,0 +1,79 @@
+import * as functions from "firebase-functions";
+import { firestore } from "firebase-admin";
+import * as mailgun from "mailgun-js";
+import { Config, isEnabled } from "./config";
+import { isAlreadyExistsError } from "./already-exists";
+import { DocumentSnapshot } from "firebase-functions/lib/providers/firestore";
+
+const config = functions.config() as Config;
+const mailgunClient = mailgun({
+  domain: config.mailgun.domain,
+  apiKey: config.mailgun.api_key,
+  host: config.mailgun.host
+});
+const db = firestore();
+
+export const EMAILS_TO_SEND_COLLECTION_NAME = "emails-to-send";
```

Comment:
Maybe "_NAME" is unnecessary ?

Reply:
I think it makes it clearer that it is a string with the name, and not a `CollectionReference` (eg `db.collection(...)`).

Reply:
So you can add a type to do this 😄 ?

Reply:
The type is inferred and collection references are immutable, so those are not taken into account in this decision. Simply, you don't see the type when you're reading the code, so having the var name explicitly tell that the content is the *name* of the collection and not the collection itself is useful, in my opinion.